app/src/main/cpp/skyline/vfs/filesystem.h

```diff
@@ -64,7 +64,7 @@ namespace skyline::vfs {
          */
         inline bool FileExists(std::string path) {
```

Comment:
This doesn't relate to this PR but these should be `const std::string&` to prevent passing by value unless the value is directly mutated in this function. There's other functions in the file which do the same, this applies to them as well.